release notes/upcoming.md

```diff
@@ -2,11 +2,10 @@ TODO: Intro
 
 ## New Features!
 
-### Category: Title (#533)
+### HTTP: request body compression (#988)
 
-Description of feature.
+Now all http methods have an additional param called `compression`, with a single possible value `gzip`, which if set will send the body compressed as gzip. It will also correctly set both `Content-Encoding` and `Content-Length`.
```

Comment:
I don't think this should be called `compression`. The HTTP header is called [`Content-Encoding`](https://tools.ietf.org/html/rfc7231#section-3.1.2.2), even though it's used primarily for compression, we probably should respect the HTTP protocol name, so `contentEncoding`? 

Or do we need to have a specific option for this in the [`http.Params`](https://docs.k6.io/docs/params-k6http) object at all? From a user-facing perspective, I'd expect that if I specify the `Content-Encoding` header, k6 will respect that. Otherwise, what would happen if the following code is executed:
```js
http.get("https://httpbin.org/anything", {
  compression: "gzip",
  headers: {
    "Content-Encoding": "deflate",
  }
});
```
ping @robingustafsson

Reply:
I kind of gave my thoughts on this in https://github.com/loadimpact/k6/pull/989#discussion_r273367020